pkg/scheduler/metrics/metrics.go

```diff
@@ -44,16 +44,25 @@ var (
 			Subsystem:      SchedulerSubsystem,
 			Name:           "schedule_attempts_total",
 			Help:           "Number of attempts to schedule pods, by the result. 'unschedulable' means a pod could not be scheduled, while 'error' means an internal scheduler problem.",
-			StabilityLevel: metrics.ALPHA,
+			StabilityLevel: metrics.STABLE,
 		}, []string{"result", "profile"})
 
 	e2eSchedulingLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:         SchedulerSubsystem,
+			Name:              "e2e_scheduling_duration_seconds",
+			DeprecatedVersion: "1.23.0",
+			Help:              "E2e scheduling latency in seconds (scheduling algorithm + binding). This metric is replaced by scheduling_duration_seconds.",
+			Buckets:           metrics.ExponentialBuckets(0.001, 2, 15),
+			StabilityLevel:    metrics.ALPHA,
+		}, []string{"result", "profile"})
+	schedulingLatency = metrics.NewHistogramVec(
 		&metrics.HistogramOpts{
 			Subsystem:      SchedulerSubsystem,
-			Name:           "e2e_scheduling_duration_seconds",
+			Name:           "scheduling_duration_seconds",
 			Help:           "E2e scheduling latency in seconds (scheduling algorithm + binding)",
```

Comment:
Don't we need to remove`E2e` from this `Help` like the above `Name:` ?

Reply:
not sure, @dgrisonnet @alculquicondor what do you think?

Reply:
Removing the word sgtm

Reply:
I think we should rename it `scheduling_attempt_seconds` or `scheduling_attempt_duration_seconds` to indicate that this measures the duration of a single attempt, failed or succeeded (it would actually be useful to break it down too by status).

This is useful to distinguish it from `pod_scheduling_duration_seconds` which includes all scheduling attempts of a pod.

Reply:
It would have to be `scheduling_attempt_duration_seconds` to adhere with naming conventions.

+1